assets/prefabs/MagicFool.prefab

```diff
@@ -39,12 +39,13 @@
           {
             "text": "How does it work?",
             "responseImage": "LightAndShadowResources:answerArrow",
-            "action": [{
-              "class": "Dialogs:NewDialogAction",
-              "target": "instructions"
-            },
+            "action": [
               {
                 "class": "Dialogs:CloseDialogAction"
+              },
+              {
+                "class": "Dialogs:NewDialogAction",
+                "target": "instructions"
               }
```

Comment:
Did you test in-game, that having two actions here actually works and has the expected effect?

Reply:
yes having two actions worked in the game. I tested the game with only one action but that only showed up the new dialog and the new responses(buttons) linked with that page did not worked so I tried adding CloseDialogAction and that closed the previous page and new page worked fine